src/bp/core/events/middleware-chain.ts

```diff
@@ -1,9 +1,8 @@
 import * as sdk from 'botpress/sdk'
+import { addStepToEvent, StepScopes, StepStatus } from 'core/events/utils'
```

Comment:
In src/bp/core/events/event-engine.ts we fetch these values directly from `./utils`. Either ways work for me, but I think we should stick with one.